style.css

```diff
@@ -402,18 +402,18 @@ textarea {
 
```

Comment:
Kindly work on these changes:
- Add the requested style.
- Check your orthographic errors.
- Add the same font family or someone similar.
- Kindly follow the style-guide from your Figma Template:
![Screen Shot 2021-08-31 at 21 33 09](https://user-images.githubusercontent.com/22258274/131602357-043b155f-e5c9-4642-b472-4a77d5d09ce9.png)